VERSION

```diff
@@ -1 +1 @@
-0.1.16
+0.1.18
```

Comment:
The version is already updated to 0.1.18 in main branch. Please do `git stash` for this file or pull the main branch.